Server/User/routers/bookings.py

```
@@ -22,6 +22,59 @@
 router = APIRouter(tags=["bookings"])
 
 
+@router.post("/consultation-tickets/return-to-inventory")
+async def return_ticket_to_inventory(
+    request: dict,
+    db: Session = Depends(get_db)
```

Ruff (0.13.3)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)